src/features/forms/components/number-form-item.tsx

```diff
@@ -42,15 +42,17 @@ export interface NumberFormItemProps<TSchema extends Record<string, unknown> = R
   decimalScale?: number
   thousandSeparator?: boolean
   placeholder?: string
+  helpText?: string | ReactElement
```

Comment:
FYI. I haven't propagated helpText to all the form items, as I was trying to constrain the number of components I touched. I can do this or we can do it as we use it?